packages/office-ui-fabric-react/src/common/sharedIsConformant.ts

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
In this case it's good to give an explanation for why the rule is being disabled (note that you specifically need to use `--` with a space on each side for eslint to accept it)
```suggestion
// eslint-disable-next-line import/no-extraneous-dependencies -- this file is for testing
```